lib/mangadex_import/chapter1.rb

```diff
@@ -0,0 +1,34 @@
+class MangadexImport
+  class Chapter1
```

Comment:
@NuckChorris I am having naming conflicts with ActiveRecord `Chapter`. Unsure how to fix